main.tf

```diff
@@ -67,3 +67,16 @@ resource "azurerm_storage_account_network_rules" "netrule" {
   bypass                     = var.traffic_bypass
 }
 
+## azure reference https://docs.microsoft.com/en-us/azure/storage/common/infrastructure-encryption-enable?tabs=portal
+## Policy ADR: https://reedelsevier.sharepoint.com/sites/OG-CoP-Cloud/SitePages/Azure-Policy---Storage-Account.aspx
```

Comment:
ADR is internal, repo is public. Please remove this comment

Reply:
right, forgot this was a public repo